creation/probes.md

```diff
@@ -169,3 +169,30 @@ filled:
 Note that in the case of Readiness Probes, it is also possible to set a
 `failureThreshold` and a `successThreashold` to only flip between ready
 and non-ready state if the probe succeeded or failed multiple times.
+
+Dual-stack considerations
+-------------------------
+
+Some context is needed to understand the limitations imposed by a dual-stack
+network configuration on readiness - or liveness - probes. Users must be
+fully aware that a dual-stack configuration is currently only available when
+using a masquerade binding type. Furthermore, it must be recalled that
+accessing a VM using masquerade binding type is performed via the pod IP
+address; in dual-stack mode, both IPv4 and IPv6 addresses can be used to reach
+the VM.
+
+Dual-stack networking configurations have a limitation when using HTTP / TCP
+probes - you **cannot probe the VMI by its IPv6 address**. The reason for this
+is the
+[HTTP](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#httpgetaction-v1-core)
+(or
+[TCP](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#tcpsocketaction-v1-core))
+probe actions `host` optional field default to the pod's IP address - which, is
+currently **always** the IPv4 address.
```

Comment:
want to add 
https://github.com/containerd/cri/issues/1278 here ?

Reply:
Not sure that's valuable information for a person attempting to configure a probe; what the user is after is how to configure the feature. 

What I could do instead is link that in the commit msg, so 'we' can track that issue and update the user guide, thus giving the user a valid, up-to-date picture.

Reply:
Done; please resolve if you think the commit msg features all the info required.

Reply:
(reviewer cant resolve)